tools/doc-generator/site-gen/build.gradle.kts

```diff
@@ -166,99 +166,87 @@ val cliHelp by tasks.registering(JavaExec::class) {
 
 val gcHelpDir = layout.buildDirectory.dir("gcHelp")
 
-val gcHelp by tasks.registering(JavaExec::class) {
-
-  inputs.files(gcRunner)
-  outputs.dir(gcHelpDir)
-
-  classpath(gcRunner)
-
-  val gcMainClass = "org.projectnessie.gc.tool.cli.CLI"
-
-  mainClass = gcMainClass
-  args("--help")
-
-  doFirst {
-    delete(gcHelpDir)
-  }
-
-  standardInput = InputStream.nullInputStream()
-  standardOutput = ByteArrayOutputStream()
+val gcHelp by tasks.registering(Sync::class) {
```

Comment:
The change in this project is to generate each help output using one Gradle task per invocation into distinct directories and then collect those using a `Sync` task

... `javaexec` is deprecated for removal in 9